### PR TITLE
#167727714 Configure Eslint

### PR DESCRIPTION
**Pull request by komsic**
#### What this PR does
This PR will lint the existing code base

#### Tasks completed
- Setup eslint
- Fix lint errors in the existing code

#### How this can be manually tested
- Clone this branch
- Change directory into it
- Run `npm install` in your terminal to install all the dependencies
- Run `npm run lint` in your terminal to lint the project

#### What are the relevant pivotal tracker stories?
[#167727714](https://www.pivotaltracker.com/story/show/167727714)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }],
+    "require-jsdoc": ["error", {
+        "require": {
+            "FunctionDeclaration": true,
+            "MethodDefinition": true,
+            "ClassDeclaration": true
+        }
+    }]
+  }
+}
```
Comment (ladaposamuel) [inline .eslintrc]: Please add a new line here.

File: src/utils/messages.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const messages ={
+const messages = {
   welcome: 'Welcome to Barefoot Nomad',
   notFound: 'Sorry, we cannot find this endpoint',
```
Comment (ladaposamuel) [inline src/utils/messages.js:3]: I think this message would sound better if its  ` The route does not exist`

**Review by ladaposamuel — CHANGES_REQUESTED** [review]

**Comment by Mcdavid95** [comment]
> * your

would it be fine to add this `./node_modules/.bin/eslint src` as an npm script instead. Makes it more accessible CC: @tejiri4

File: .eslintrc (hunk 1 above)
Comment (komsic) [reply .eslintrc]: Thanks for the review. I will soon add the line

File: src/utils/messages.js (hunk 2 above)
Comment (komsic) [reply src/utils/messages.js:3]: I am not the original writer of the message. I only changed the `CRLF` on the line to `LF`. Do you still think I should change it?

**Comment by komsic** [comment]
> > * your
> 
> would it be fine to add this `./node_modules/.bin/eslint src` as an npm script instead. Makes it more accessible CC: @tejiri4

Thanks for the review. I was not thinking of accessibility. Will create an npm script for it soon.

File: .eslintrc (hunk 1 above)
Comment (komsic) [reply .eslintrc]: Done. Please resolve the conversation

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+src/database
```
Comment (Mcdavid95) [inline .eslintignore:2]: I think the `dist` folder generated by babel should be here too

**Review by Mcdavid95 — CHANGES_REQUESTED** [review]

File: .eslintignore (hunk 3 above)
Comment (komsic) [reply .eslintignore:2]: Done. Please resolve the conversation.

File: src/utils/messages.js (hunk 2 above)
Comment (ladaposamuel) [reply src/utils/messages.js:3]: You may leave it for now since it's just a dummy text

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (tejiri4) [inline .eslintrc]: currently using `var` is not flag by eslint rename this file from `.eslintrc` to `.eslintrc.json` and also correct the config_file details in `.hound.yml`

Comment (komsic) [reply .eslintrc]: Alright

Comment (komsic) [reply .eslintrc]: Done. Please resolve the conversation.

**Review by tejiri4 — APPROVED** [review]
Nice work LGTM

**Pull request merged**